visualpic/data_reading/particle_readers.py

```diff
@@ -75,9 +75,12 @@ def _read_component_metadata(self, file_path, species, component):
             metadata['time']['units'] = self._numpy_bytes_to_string(
                 file_handle.attrs['TIME UNITS'][0])
             metadata['grid'] = {}
-            metadata['grid']['resolution'] = file_handle.attrs['NX']
-            max_range = file_handle.attrs['XMAX']
-            min_range = file_handle.attrs['XMIN']
+            simdata_path = '/SIMULATION'
```

Comment:
This is the fix for the new Osiris versions, right? Should this change also be applied to the field readers?